rca/scholarships/views.py

```diff
@@ -45,3 +56,40 @@ def scholarships_delete(request):
             "submit_url": (reverse("scholarships_delete")),
         },
     )
+
+
+def load_scholarships(request):
+    scholarships = Scholarship.objects.all()
+
+    try:
+        programme = ProgrammePage.objects.get(pk=int(request.GET.get("programme")))
```

Comment:
I think we will need this to work with slugs rather than the IDs. So links are descriptive when sent to students, EG enquire?programme=architecture

Reply:
Done. It needs to work with both otherwise lots of form changes required...